_15thnight/api/account.py

```diff
@@ -71,12 +70,15 @@ def forgot_password():
         return api_error(form.errors)
     user = User.get_by_email(form.email.data)
     if user:
-        if (not user.reset_token or
-            user.reset_created_at < datetime.utcnow() - reset_token_life):
```

Comment:
If you feel compelled to change the formatting here, please just indent the wrapped if-statement in an extra 4 spaces. It will hit 79 perfectly.